terraform/transform_orphan_count.go

```diff
@@ -0,0 +1,110 @@
+package terraform
+
+import (
+	"log"
+
+	"github.com/hashicorp/terraform/dag"
+)
+
+// OrphanResourceCountTransformer is a GraphTransformer that adds orphans
+// for an expanded count to the graph. The determination of this depends
+// on the count argument given.
+//
+// Orphans are found by comparing the count to what is found in the state.
+// This tranform assumes that if an element in the state is within the count
```

Comment:
seriously, is the "tranform" typo the only thing fix you're going to throw me ;)